lib/Twig/Environment.php

```diff
@@ -1168,7 +1168,7 @@ protected function initExtension(Twig_ExtensionInterface $extension)
         }
     }
 
-    protected function writeCacheFile($file, $content)
+    private function writeCacheFile($file, $content)
```

Comment:
we have a recipe in the doc overwriting this method. You should either keep this as an extension point or remove the documentation about using this method as an extension point